airflow/providers/oracle/hooks/oracle.py

```diff
@@ -52,6 +52,8 @@ class OracleHook(DbApiHook):
 
     supports_autocommit = True
 
+    _test_connection_sql = "select 1 from dual"
```

Comment:
Unfortunately we can’t do this yet because the provider still needs to support Airflow 2.2 and lower. The full `test_connection` function still needs to exist for now.